main.py

```diff
@@ -0,0 +1,47 @@
+from rubikState import RubikState, Moves
+from thistlethwaiteAlgo import solver
+from cubiesMoves import cubiesMove
+from mixParsing import getMix, convert_for_3D
+from showcase3D import showcase
+
+import sys
+import cProfile
+
+def main():
+
+    if (len(sys.argv) != 2):
+        sys.exit('Wrong number of arguments')
+
+    input = sys.argv[1]
+    faceletsMix = getMix(input)
+    cubiesMix = input.split()
+
+    rubik = RubikState()
+
+    for one_move in faceletsMix:
```

Comment:
Why not just `move` ?

Reply:
at some point while programming i had issue because move is an existing function so i just changed everything for one_move to avoid such pb, not very pretty but it works fine ^^

Reply:
Ohh, ok no prob then!